### ISSUE TITLE
Waarom gebruik je font-awesome?

### ISSUE THREAD

**Issue by kriskuiper** [muhammet075/FrontendDev#5]
Ik denk dat er een nettere manier is om icoontjes te gebruiken, bijvoorbeeld door een svg op te zoeken en die te gebruiken. Font-awesome is redelijk deprecated tegenwoordig.

https://github.com/muhammet075/FrontendDev/blob/dcc36bc391386c5af4c26c591dc52ff9240378b0/index.html#L31-L32

**Comment by muhammet075** [comment]
oh, dat wist ik niet ik dacht dat font awesome juist een goede manier was om icoontjes te gebruiken

**Comment by kriskuiper** [comment]
Nee, dat is 't totaal niet. Schermlezers kunnen die icoontjes spaarzaam of helemaal niet lezen waardoor je website dus veel minder accessible wordt
